pkg/scanners/scannerv4/convert.go

```diff
@@ -68,6 +68,24 @@ func layerIndex(layerSHAToIndex map[string]int32, report *v4.VulnerabilityReport
 	return nil
 }
 
+func sourceType(pkg *v4.Package) storage.SourceType {
+	db := pkg.GetPackageDb()
+	switch {
```

Comment:
I have a bit of a preference to parse the db portion out and split by the :, then don't need to have hasprefix so many times and can just check the value directly

Reply:
done